cfn-resources/profile/profile.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package profile
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+)
+
+const (
+	defaultProfile    = "default"
+	profileNamePrefix = "cfn/atlas/profile"
+)
+
+type Profile struct {
+	PublicKey  string `json:"PublicKey"`
+	PrivateKey string `json:"PrivateKey"`
+	BaseURL    string `json:"BaseUrl"`
```

Comment:
using the same as terraform

Reply:
Should we ignore nulls/unset baseURL?

```suggestion
	BaseURL    string `json:"BaseUrl, omitempty"`
```